config/prod/test-amazonlinux2.yaml

```diff
@@ -0,0 +1,21 @@
+# Provision an EC2 instance
+template_path: templates/managed-ec2-v2.yaml
+stack_name: amazon-linux2
+parameters:
+  # The Sage deparment for this resource
+  Department: "CompOnc"
+  # The Sage project this resource will be used for
+  Project: "Amazon linux 2 base image"
+  # The resource owner
+  OwnerEmail: "thomas.yu@sagebase.org"
+  # EC2 instance type (available types https://aws.amazon.com/ec2/instance-types/)
+  InstanceType: "t2.micro"
+  AMIId: "ami-056131b84aa59cae0"
```

Comment:
this AMI's disk is NOT encrypted, /dev/xvda=snap-0696504305c0108ce:8:true:gp2

lets test an AMI with an encrypted disk.

Reply:
Test with VolumeSize parameter as well?